### ISSUE TITLE
[CYPRESS] Add e2e test for PayPal wallet (redirection flow) payment method for PayPal Connector

### ISSUE THREAD

**Issue by Sanskar2001** [juspay/hyperswitch#6044]
The [Hyperswitch SDK](https://docs.hyperswitch.io/hyperswitch-cloud/integration-guide) is a versatile tool built to streamline payment integration for developers. It simplifies the process of adding multiple payment methods, allowing you to quickly implement support for various gateways without the need to manually integrate each one. With Hyperswitch, you can efficiently manage payments across platforms, reducing complexity and boosting development speed.

Pre-requisites:
Please follow the [README](https://github.com/juspay/hyperswitch-web/blob/main/cypress-tests/readme.md) for guidelines on file structure, formatting and instructions on how to set up and run Cypress.

### **Description**:

This task involves adding a cypress e2e test for PayPal wallet (redirection flow) for PayPal Connector.

**🔨 Possible Implementation**
Locate the element for this payment method, perform the click action on it and assert whether redirection happens success.

## Contribution Guidelines:

- Fork the repository and create a new branch for your work.
- Ensure the WebSDK follows best practices for API integration and field rendering.
- Write clean, well-documented code with clear commit messages.
- Make sure to follow our coding standards and contribution guidelines.

---

## Helpful Resources:

- Link to WebSDK documentation: [WebSDK Docs](https://docs.hyperswitch.io/hyperswitch-cloud/integration-guide)
- Link to the API endpoint specifications: [API Docs](https://api-reference.hyperswitch.io/introduction)

### Submission Process:

- Ask the maintainers for assignment of the issue, you can request for assignment by commenting on the issue itself.
- Once assigned, submit a pull request (PR).
- Maintainers will review and provide feedback, if any.
- Maintainers can unassign issues due to inactivity, [read more here](https://github.com/juspay/hyperswitch/wiki/Hacktoberfest-Contribution-Rules).

Refer [here](https://github.com/juspay/hyperswitch/blob/main/docs/TERMS_OF_CONTEST.md) for Terms and conditions for the contest.

If you have any questions or need help getting started, feel free to ask in the comments!

**🔖 Note:**
Please drop a mail for required credentials at preetam.revankar@juspay.in

**Comment by CODEX108** [comment]
Hey @Sanskar2001 @gorakhnathy7 can you please assign me the issue?

**Comment by gorakhnathy7** [comment]
Hey @CODEX108 Assigned the issue to you!
Thanks for your interest!

**Comment by CODEX108** [comment]
Hello @gorakhnathy7 thank you for assigning the issue!
I have a quick question regarding the E2E test setup for the PayPal wallet (redirection flow) payment method. Should I be writing the tests in the cypress/e2e folder? Could you also clarify the exact folder where I should place the test cases?

Thanks in advance!

**Comment by Sanskar2001** [comment]
Hi @CODEX108 
Yes, you need to place your tests in cypress/e2e folder, basically your tests should be placed here https://github.com/juspay/hyperswitch-web/tree/main/cypress-tests/cypress/e2e. 


You only need payments-create body, publishable key and  secret key to complete this task. The credentials will be shared with you over mail.

**Comment by CODEX108** [comment]
Okay Thank you @Sanskar2001 !

**Comment by gorakhnathy7** [comment]
Hey @CODEX108 
- For this issue, please submit a PR on [@juspay/hyperswitch-web](https://github.com/juspay/hyperswitch-web) repo, and link it to the issue.

**Comment by gorakhnathy7** [comment]
Hey @CODEX108 ,

Quick check-in on the status of this issue. If you have any questions or need assistance, feel free to reach out in the [community](https://join.slack.com/t/hyperswitch-io/shared_invite/zt-2jqxmpsbm-WXUENx022HjNEy~Ark7Orw).

**Comment by CODEX108** [comment]
> Hey @CODEX108 ,
> 
> Quick check-in on the status of this issue. If you have any questions or need assistance, feel free to reach out in the [community](https://join.slack.com/t/hyperswitch-io/shared_invite/zt-2jqxmpsbm-WXUENx022HjNEy~Ark7Orw).

Yes I've asked about the issue on `dev-support`,since there are a  few errors while running the app.

**Comment by gorakhnathy7** [comment]
Hey @CODEX108 

Any update on the status of the issue?

**Comment by CODEX108** [comment]
Hey please unassign the PR, I wasn't able to complete the task successfully.

**Comment by AKC4112002** [comment]
Hi! I'd like to work on this issue as a beginner contributor. Please assign it to me if possible.
